coversation.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env python2
+
+import time
+from naoqi import ALProxy
+
+
+def repeat_test():
+    '''
+    test code - pepper repeats word back to user
+    '''
+    IP = "whatever the IP is"
+    tts = ALProxy("ALTextToSpeech", IP, 9559)  # initiates text to speech functionality
+    asr = ALProxy("ALSpeechRecognition", IP, 9559)  # initiates speech recognition functionality
+    memory = ALProxy("ALMemory")
+
+    asr.setlanguage("English")
+    vocab = ["good", "bad"]
+    asr.setVocabulary(vocab, False)  # sets what pepper understands

```

Comment:
I did not understand what if we set `asr.setVocabulary(vocab, True)`? I did not understand what does **word spotting** described [here](https://developer.softbankrobotics.com/pepper-naoqi-25/naoqi-developer-guide/naoqi-apis/naoqi-audio/alspeechrecognition-1#-adding-words-to-the-vocabulary--sbr-fake-anchor) means. Do you have any idea?

Reply:
I don't fully understand the feature either, there is a little bit more information on it [here](http://doc.aldebaran.com/2-1/naoqi/audio/alspeechrecognition.html) but to me that looks like the phrases will be easier to process with word spotting disabled as we will only have small predefined user answers.

Reply:
All right. We can try first.